.codeclimate.yml

```diff
@@ -35,6 +47,10 @@ engines:
         enabled: false
       LeadingZero:
         enabled: false
+      HexLength:
+        enabled: false
+      PseudoElement:
```

Comment:
The difference is merely for us humans. All browsers treat the pseudo selector/attribute with the single colon. I think we should conform to this standard as it is the new standard.